cli/cli_test.go

```diff
@@ -210,60 +210,67 @@ func Example_ranges() {
 	c.Run("kv scan")
 	c.Run("quit")
 
-	// Output:
```

Comment:
The comment `Output:` is special for example tests; it needs to be there to trigger the check that the output matches.